virttest/env_process.py

```diff
@@ -1011,15 +1011,24 @@ def preprocess(test, params, env):
             test.cancel("Failed to map hostname and ipaddress of target host")
         session.close()
 
-    # Destroy and remove VMs that are no longer needed in the environment
+    # Destroy and remove VMs that are no longer needed in the environment or
+    # leave them untouched if they have to be disregarded only for this test
     requested_vms = params.objects("vms")
+    vm_env_action = params.get("vm_env_action", "destroy")
```

Comment:
How does the name `vm_disposal` sound like?

Reply:
Maybe `unrequested_vm_disposal` is more accurate.

Reply:
Given your point about the `env_cleanup` parameter and the fact that we won't use `vm_env_action` for this purpose too, indeed the name is no longer the most precise and should be changed. Let's choose `unrequested_vms_disposal` as a boolean `yes` and `no` parameter then. Coming with the next push.

Reply:
IMO the previous choices of `vm_env_action` were a bit more future proof for multiple possibilities but this is not that important and maybe it won't be extended any further.

Reply:
I am resolving this for the sake of my summary of anything left from these reviews below.